ci/release/changelogs/next.md

```diff
@@ -2,9 +2,11 @@
 
 - Tooltips can be set on shapes. See [https://d2lang.com/tour/tooltips](https://d2lang.com/tour/interactive). [#548](https://github.com/terrastruct/d2/pull/548)
 - Links can be set on shapes. See [https://d2lang.com/tour/tooltips](https://d2lang.com/tour/interactive). [#548](https://github.com/terrastruct/d2/pull/548)
+- The `width` and `height` attributes are no longer restricted to images and can be applied to non-container shapes. [#498](https://github.com/terrastruct/d2/pull/498)
```

Comment:
> non-container shapes

Maybe, _non-container objects_ as there're no such thing as container shapes

Reply:
i wanted to call every "node" a shape, as object is pretty all-encompassing. but i'm not 100% sure about that route

Reply:
well, from a changeling perspective, shape reminds me of `shape` keyword and not the objects themselves

Reply:
<img width="946" alt="Screen Shot 2022-12-29 at 12 07 15 PM" src="https://user-images.githubusercontent.com/85081687/210006385-addddcec-2941-46bc-9d17-daa625c5074f.png">